pkg/cmd/cli/install/install.go

```diff
@@ -130,6 +134,38 @@ func NewCommand(f client.Factory) *cobra.Command {
 	return c
 }
 
+func GetVeleroVersion(f client.Factory) (string, error) {
+	// Retrieve velero version on server side
+	serverStatusGetter := &serverstatus.DefaultServerStatusGetter{
+		Namespace: f.Namespace(),
+		Timeout:   5 * time.Second,
+	}
+	var veleroClient velerov1client.ServerStatusRequestsGetter
+	client_v, err := f.Client()
+	cmd.CheckError(err)
+	veleroClient = client_v.VeleroV1()
+	serverStatus, err := serverStatusGetter.GetServerStatus(veleroClient)
+	if err != nil {
+		return "", err
+	}
+	version := serverStatus.Status.ServerVersion
+    return version, nil
+}
+
+func CheckVeleroVersion(version string) bool {
```

Comment:
We need constants for expected version.  The way you have this code, everytime we change the version we need we have to change 8 different places in the code that I count.

Let's have
MinVeleroMajorVersion ("1")
MinVeleroMinorVersion ("3")
MinVeleroPointVersion ("1" currently)

Reply:
Add a general CompareVersion(currentVersion string, minVersion string) to make it easy to update the min version. In another change: check csi version will use the same method.